app/assets/stylesheets/modules/homepage.scss

```diff
@@ -2,14 +2,13 @@
 @import "mixins";
 
 body.home-page {
-  background: url(dog.png) no-repeat;
-  background-size: 100%;
+  background: url(dogstay.png)no-repeat fixed;
+  background-size:cover;
```

Comment:
Properties should be ordered background, background-position, background-size, opacity<br>Colon after property should be followed by one space